scripts/opensearch-scripts/get-rank-aggregated-ctr.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
```

Comment:
Just a query to easily look for zeroes.

Reply:
When this query is meant to look for zeroes shouldn't it look like this (in Dev Tools style):

```
GET rank_aggregated_ctr/_search
{
  "query": {
    "range": {
      "ctr": {
        "gte": 0,
        "lte": 0
      }
    }
  }
}
```

Reply:
Oh, sorry. Meant to look for things that are not zeroes. Since everything was 0, I wanted to look at the > 0 ones afterward.

Reply:
ok, that makes sense then, thanks!